docker_reverse_proxy/conf/reverse-proxy.conf.template

```diff
@@ -0,0 +1,38 @@
+upstream relay {
+    server sentry_relay.sentry:${RELAY_PORT};
```

Comment:
We should probably have RELAY_HOST variable too?

Reply:
no I think sentry_relay is reasonable, we use SENTRY_HOST to be able to inject host.docker.internal or localhost depending on the OS .